Tests/Fixtures/Generated/Colors/swift3-context-defaults-publicAccess.swift

```diff
@@ -37,7 +37,7 @@ public struct ColorName {
 // MARK: - Implementation Details
 
 // swiftlint:disable operator_usage_whitespace
-public extension Color {
+internal extension Color {
```

Comment:
Shouldn't that still be `public` as this is the output of when you use the `publicAccess: true` flag?

(Note: I hope you didn't lose too much time editing the generated fixtures one by one by hand 😅  If you missed it, there's a way to automatically update all of those Generated fixtures using a dedicated Scheme, see `CONTRIBUTING.md` for more details)

Reply:
I guess I could go either way on that, and would defer to your judgment. I changed those to internal based on @djbe's comment here: https://github.com/SwiftGen/SwiftGen/issues/548#issuecomment-427953553.

And I didn't see there was a way to generate those, but luckily it wasn't too much work to manually update them 😄

Reply:
Ah, right, I missed that your PR fixed actually two things:
 - missing `public` on some lines when you use `publicAccess: true`
 - too much useless annotations on some lines (even in the public case) that shouldn't be annotated as public

TBH that comment was just a quick one after a super-quick glance at the diff, so I probably overlooked that, will take a closer look this evening :wink:

Reply:
Whoops, that's my fault. I forgot to mention the second fix in the PR. I'll add that to the changelog and PR description.

Thanks for looking it over and let me know if there are any changes you'd like me to make 👍

Reply:
This has been updated in 8e15873